src/microservices/Microservices.DicomTagReader/Execution/DicomTagReaderHost.cs

```diff
@@ -18,21 +19,27 @@ public class DicomTagReaderHost : MicroserviceHost
         public DicomTagReaderHost(GlobalOptions options)
             : base(options)
         {
+            // We don't generally want tag validation in the SMI pipeline: can't fix upstream data at this stage!
+            new DicomSetupBuilder().SkipValidation().Build();
```

Comment:
@jas88  Do we need this in DicomReprocessor and other services too? should we move it to the base class for microservice hosts?

Reply:
Good point - I've replaced all but one of the DicomSetupBuilder calls with one in the Microservice host constructor; the IdentifierMapper unit tests needed a call of their own.